enhancements/cluster-logging/cluster-logging-otel-support.md

```diff
@@ -0,0 +1,240 @@
+
+**cluster-logging-operator** is an operator responsible for managing log collection and forwarding.
+
+The cluster administrator does the following:
+
+1. Deploys the Red Hat **cluster-observability-operator**
+1. Deploys the Red Hat **loki-operator** which supports server-side indexing and OTLP ingestion endoint
+1. Deploys an instance of **LokiStack** into the `openshift-logging` namespace
+1. Deploys the Red Hat **cluster-logging-operator**
+1. Creates a **ClusterLogForwarder** custom resource for the **LokiStack**
+    1. **NOTE:** In Tech Preview, additionally annotated to instruct the operator to use the OTEL datamodel in lieu of ViaQ
+
+The **cluster-observability-operator**:
+
+1. Deploys the console logging-view-plugin for reading logs in the OpenShift console
+
+The **loki-operator**
+
+1. Deploys the **LokiStack** for storing logs in-cluster
+
+The **cluster-logging-operator**:
+
+1. Deploys the log collector to forward logs to log storage in the `openshift-logging` namespace
+
+
+### API Extensions
+
+During the time this feature is considered Tech Preview, instances of **ClusterLogForwarder** will be annotated to indicate to the
+operator that logs will be forwarded to **LokiStack** using OTLP:
+
+```
+   observability.openshift.io/enable-lokistack-otlp: [[LOKISTACK_NAMES]]
+```
+* LOKISTACK_NAMES := Comma delimited list of **clusterlogforwarder.spec.output[].name** of type **lokiStack**
+
+Example of configuring multiple outputs:
+
+```yaml
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  annotations:
+    logstore.observability.openshift.io/otlp: my-loki-otel,my-other-loki-otel
+spec:
+  outputs:
+  - name: my-loki-otel
+    type: lokiStack
+  - name: my-other-loki-otel
+    type: lokiStack
+  - name: my-legacy-loki
+    type: lokiStack
+  pipelines:
+  - name: in-to-out
+    outputRefs: [my-loki-otel,my-other-loki-otel,my-legacy-loki]
+    inputRefs: [application]
+```
+
+This example configures a collector to forwards logs to multiple LokiStacks (e.g. my-loki-otel,my-other-loki-otel) using OTEL semantic attributes while
+also forwarding logs to a separate LokiStack (e.g. my-legacy-loki) using ViaQ.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+#### Standalone Clusters
+
+#### Single-node Deployments or MicroShift
+
+### Implementation Details/Notes/Constraints
+
+#### Log Storage
+
+* The **loki-operator** will be updated so deployments of Lokistack rely upon the OTLP API instead of the Push API. This will allow
+trusted and verified third-party Red Hat products, like Red Hat Build of the OpenTelemetry Collector, to forward logs to the same LokiStack as the Red Hat OpenShift Logging.  Additionally, this new version of the **loki-operator**
+will support server-side indexing to improve query performance and simplify configuration on the client side. Sane defaults will be provided for users using the `openshift-*` tenancies.
```

Comment:
Currently I am under the impression that we're only providing OTLP defaults for logging and not for the network observability operator?

```suggestion
will support server-side indexing to improve query performance and simplify configuration on the client side. Sane defaults will be provided for users using the `openshift-logging` tenancy.
```

Reply:
> Currently I am under the impression that we're only providing OTLP defaults for logging and not for the network observability operator?

@xperimental who can answer this question and what changes need to be made to the document?

Reply:
Determined replacing "tenancy" to be only "openshift-logging" addresses this comment and my follow-up question.